challenges/day5-issues/url-shortener/frontend/index.html

```
@@ -0,0 +1,22 @@
+# URL Shortener (Frontend)
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)